packages/Path-View-Core.package/SClassDiagram.class/instance/boundingBoxOfClassMorphs.st

```diff
@@ -0,0 +1,10 @@
+geometry
+boundingBoxOfClassMorphs
+	"calculate the bounding box of all class morph in the view"
+
+	| emptyRectangle |
+	emptyRectangle := 0@0 corner: 0@0.
+	^ self classMorphs 
```

Comment:
does someone know which of both is more common in the project? I would prefer less work, though I tend to like the `self` syntax more